packages/pwa-kit-cli/src/ssr/server/build-dev-server.js

```diff
@@ -153,7 +153,7 @@ export const DevServerMixin = {
             if (app.__webpackReady()) {
                 middleware(req, res, next)
             } else {
-                res.redirect(301, '/__mrt/loading-screen/index.html?loading=1')
+                this._onAllRequestsBeforeWebpackReady(req, res, next)
```

Comment:
@bendvc I see now how the previously the function `_useWebpackHotServerMiddleware` isn't accurate. How about this instead?

What I actually wanted is to extract out the piece that does the redirect for those requests before webpack build is ready.